src/helpers/findByAPI.js

```diff
@@ -32,34 +37,68 @@ export const findAllByTestId = (instance: ReactTestInstance) => (
 
 export const findByText = (instance: ReactTestInstance) => (
   text: string | RegExp,
-  waitForOptions: WaitForOptions = {}
-) => makeFindQuery(instance, getByText, text, waitForOptions);
+  waitForOptions: WaitForOptions = {},
+  textMatchOptions?: TextMatchOptions
+) => makeFindQuery(instance, getByText, text, waitForOptions, textMatchOptions);
 
 export const findAllByText = (instance: ReactTestInstance) => (
   text: string | RegExp,
-  waitForOptions: WaitForOptions = {}
-) => makeFindQuery(instance, getAllByText, text, waitForOptions);
+  waitForOptions: WaitForOptions = {},
+  textMatchOptions?: TextMatchOptions
+) =>
+  makeFindQuery(instance, getAllByText, text, waitForOptions, textMatchOptions);
 
 export const findByPlaceholderText = (instance: ReactTestInstance) => (
   placeholder: string | RegExp,
-  waitForOptions: WaitForOptions = {}
-) => makeFindQuery(instance, getByPlaceholderText, placeholder, waitForOptions);
+  waitForOptions: WaitForOptions = {},
+  textMatchOptions?: TextMatchOptions
+) =>
+  makeFindQuery(
+    instance,
+    getByPlaceholderText,
+    placeholder,
+    waitForOptions,
+    textMatchOptions
+  );
 
 export const findAllByPlaceholderText = (instance: ReactTestInstance) => (
   placeholder: string | RegExp,
-  waitForOptions: WaitForOptions = {}
+  waitForOptions: WaitForOptions = {},
+  textMatchOptions?: TextMatchOptions
```

Comment:
```suggestion
  queryOptions?: TextMatchOptions
```
Consider naming textMatchOptions as queryOptions it match RTL docs (please change in other places as well).